upgrade/satellite.py

```diff
@@ -120,6 +120,15 @@ def satellite6_upgrade(zstream=False):
     disable_repo_name = ["*"]
     enable_repos_name = ['rhel-{0}-server-rpms'.format(major_ver),
                          'rhel-server-rhscl-{0}-rpms'.format(major_ver)]
+
+    # This statement will execute only until downstream release not become beta.
+    if os.environ.get('NON_BETA_FM_UPGRADE') == 'true' or \
+            os.environ.get('PERFORM_FOREMAN_MAINTAIN_UPGRADE') == 'false':
```

Comment:
No its not required here, it has been in below if condition.

My comment was to check if enabling of disable repos handled somewhere below and remove it from here. But I think its not as per ur last coment.

Reply:
I think your comment was valid, we call  "enable_disable_repo(disable_repo_name, enable_repos_name)" function two times one for downstream-fm upgrade and other for the non-fm upgrade, here we have handled both in one condition.